src/components/app.tsx

```diff
@@ -10,8 +10,43 @@ import {EncodingPane} from './encoding-pane';
 import {Header} from './header';
 import {ViewPane} from './view-pane';
 
+import {
+  ActionHandler,
+  DatasetAsyncAction,
+  datasetUrlLoad,
+} from '../actions';
+
+
+
+interface DataURL {
+  name: string;
+  url: string;
+};
+export type VoyagerData = DataURL | Object[];
```

Comment:
Oh interesting. Didn't know about the vega-lite data object. Since doing this i've been thinking of what a consistent format could be and started talking about that in #324, basically `{ "type": "dataset", name: "Foo", "values": [] }`. But i'll look at that interface/type in vega-lite and see if that can suffice.